src/app/todos/todo.model.ts

```diff
@@ -0,0 +1,18 @@
+export enum TodoStatus {
+  Backlog = 'backlog',
+  InProgress = 'in-progress',
+  Done = 'done',
+  None = '',
+}
+
+export class Todo {
+  public id: string;
+  public title: string;
+  public status: TodoStatus;
+
+  constructor(id: string, title: string, status: TodoStatus) {
+    this.id = id;
+    this.title = title;
+    this.status = status;
+  }
+}
```

Comment:
I would like Pavel's input on this one as well. I personally do not like using classes for anything except components themselves. It's always better to create an interface and then create an object following that interface without calling a constructor function

Reply:
I agree with Alex